Gemfile

```diff
@@ -0,0 +1,53 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.6.5'
+
+gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+
+gem 'mysql2', '~> 1.4'
+# Use Puma as the app server
+gem 'puma', '~> 4.1'
+# Use SCSS for stylesheets
+gem 'sass-rails', '>= 6'
+# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
+gem 'webpacker', '~> 4.0'
```

Comment:
remove all the comment lines, using " instead of '

Reply:
Done

Reply:
fix it, please. Follow this document
https://docs.google.com/document/d/104Csp4-vamVos5DEbi372nLBEfmqhj7h1ENYO8ebjCo/edit